Makefile.inc1

```diff
@@ -571,7 +571,6 @@ VERSION=	FreeBSD ${_REVISION}-${_BRANCH:C/-p[0-9]+$//} ${TARGET_ARCH} ${SRCRELDA
 .endif
 MAJOR_REVISION=	${_REVISION:R}
 
-.if !defined(PKG_VERSION)
```

Comment:
The below variables should always be defined regardless of what PKG_VERSION is set to, since it is used to compare two repos for incremental updates.